apps/dashboard/src/pages/Root.tsx

```diff
@@ -1,8 +1,24 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { Toaster } from "@litespace/luna";
-import { Outlet } from "react-router-dom";
+import { Outlet, useNavigate } from "react-router-dom";
+import { useAppSelector } from "@/redux/store";
+import { profileSelectors } from "@/redux/user/profile";
+import { IUser } from "@litespace/types";
+import { Route } from "@/lib/route";
 
 const Root: React.FC = () => {
+  const profile = useAppSelector(profileSelectors.user);
+  const navigate = useNavigate();
```

Comment:
Please add empty line. 
```suggestion
  const navigate = useNavigate();
  
```